src/patcher.py

```diff
@@ -8,6 +8,7 @@
 from parse.stage import StagefileManager
 from util.file_copy import clean_scripts, copy_file, get_decomp_locations
 
+# pylint: disable=pointless-string-statement
```

Comment:
Where is this an issue?

Reply:
Since this is the only "pointless string statement" Pylint warning in the entire program, I have opted to suppress it here rather than doing so globally. This should allow us to keep this comment, while still catching all other Pylint warnings of the same type.